lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -590,6 +632,209 @@ class HTMLInputElementImpl extends HTMLElementImpl {
     return null;
   }
 
+  get _parsedValue() {
+    const converter = this._convertStringToNumber;
+    if (converter !== undefined) {
+      return converter(this.value);
+    }
+    return this.value;
+  }
+
+  // https://html.spec.whatwg.org/multipage/input.html#attr-input-step
+  get _step() {
+    let step = this._defaultStep;
+    if (this.hasAttribute("step") && this.getAttribute("step").toLowerCase() !== "any") {
```

Comment:
Don't use toLowerCase here; instead use the ASCII case-insensitive version in strings.js

Reply:
done